locale/es.lua

```diff
@@ -7,19 +7,26 @@ local Translations = {
         no_access = 'No tienes acceso a este comando!',
         company_too_poor = 'Tu empresa está en bancarrota. No hay dinero suficiente.',
         item_not_exist = 'El objeto no existe en el sistema.',
-        too_heavy = 'Tienes el inventario muy lleno.'
+        too_heavy = 'Tienes el inventario muy lleno.',
+        duplicate_license = 'Aun estas dentro.',
```

Comment:
This text could leave to a bad interpretation, a better one will be  "Ya existe un Jugador con esta licencia dentro del servidor", or something like that